src/rantanplan/rhymes.py

```diff
@@ -28,7 +32,7 @@
     ("v", "b"), ("ll", "y"),
     ("ze", "ce"), ("zi", "ci"),
     ("qui", "ki"), ("que", "ke"),
-    ("ge", "je"), ("gi", "ji"),
+    ("ge", "je"), ("gi", "ji")
```

Comment:
:thinking:

Reply:
Sorry I was testing stuff and forgot to remove that.

Reply:
The line needs to be there though. I was wondering about the trailing comma.
```
 ("ge", "je"), ("gi", "ji"),
```